ivy.el

```diff
@@ -229,7 +229,9 @@ ACTIONS that have the same key."
              ("o" identity "default")
              ,@extra-actions))
           (t
-           (delete-dups (append action extra-actions))))))
+           `(1
+             ,@(cl-delete-duplicates (cdr (append action extra-actions))
+                                   :key #'car :test #'equal :from-end t))))))
```

Comment:
Nit: Indentation seems off here.

Reply:
Alright, I followed the instructions found [here](https://github.com/abo-abo/swiper/blob/master/CONTRIBUTING.org#contributing-code). I might have messed something up though. What would you expect to see? 

I'm seeing a couple of similar patterns in the code (notably at 211-215 and 219-220). It seems the tail `:key #'car ...` wants to be aligned to the same position that the first argument of the function. 

For example, if I move `(cdr (append ...` to the next line I get:
```emacs-lisp
`(1
  ,@(cl-delete-duplicates
     (cdr (append action extra-actions))
     :key #'car :test #'equal :from-end t))
```

Reply:
Ahh, now I realize that the indentation above does not match what I just described :sweat: Ignore me! I'll get it updated w/ the right indentation, thanks!